core/domain/feedback_jobs_continuous.py

```diff
@@ -63,7 +63,7 @@ def _handle_incoming_event(cls, active_realtime_layer, event_type, *args):
             active_realtime_layer: int. The currently active realtime
                 datastore layer.
             event_type: str. The event triggered by the student.
-            *args: Variable length argument list. The first element of *args
+            args: variable_length_argument_list. The first element of *args
```

Comment:
args: list(*). Variable length argument list. ...

(Seems like a good idea to keep the * too for parity with method signature?)

Reply:
Done, the * issue too.